templates/stamp/utilities.sh

```diff
@@ -632,10 +632,23 @@ install-azure-cli()
         apt-get -y -qq update
 
         # Note: nodejs-legacy is required for Ubuntu14 and above.
-        log "Installing nodejs-legacy, npm, and azure cli"
-        apt-get install -y nodejs-legacy npm
-        exit_on_error "Failed to install nodejs-legacy and/or npm on ${HOSTNAME} !" $ERROR_NODEINSTALL_FAILED
+        short_release_number=`lsb_release -sr`
+        if [[ "$short_release_number" > 13 ]]; then
```

Comment:
I recall a conversation about this. Is this still necessary for Ubuntu 14.*? The nodejs-legacy is the source of some issues affecting azure cli.